File: countwords.py

def count_words(words):
    counts = {}
    
    for word in words:
        counts[word] = counts.get(word, 0) + 1
    
    return counts

# Example
print(count_words(['apple', 'grape', 'apple', 'apple']))
# Output: {'apple': 3, 'grape': 1}

File: countwords.js

function countWords(words) {
    const counts = {};
    
    for (let word of words) {
        counts[word] = (counts[word] || 0) + 1;
    }
    
    return counts;
}

// Example
console.log(countWords(['apple', 'mango', 'pear', 'mango', 'mango', 'pear']));
// Output: { apple: 3, grape: 1 }